frontend/src/util/ws_mousetrap_fork.js

```diff
@@ -20,7 +20,7 @@
  * @version 1.6.3
  * @url craig.is/killing/mice
  */
-(function(window, document, undefined) {
+(function(window, document) {
```

Comment:
I think this change broke something. Now, if I use j / k to scroll down a worksheet list (subworksheets block) and press enter on one of them, I get this error:

![image](https://user-images.githubusercontent.com/1689183/94934694-12187700-049a-11eb-9094-eaee8071ebe6.png)


It's better to just leave this file as-is and then just include ws_mousetrap_fork in the eslintrc so it's ignored by eslint, since this is external code, anyway.

Reply:
Thanks for pointing it out. Just updated it.

Reply:
Actually, this error appears to be happening on master, and is unrelated to these changes, so will make a new issue for it.